src/util/images.tsx

```diff
@@ -2,7 +2,7 @@ import { LxdImage, RemoteImage } from "types/image";
 import { LxdStorageVolume } from "types/storage";
 
 export const isVmOnlyImage = (image: RemoteImage) => {
-  return image.variant?.includes("desktop") || image.server === LOCAL_ISO;
+  return image.variant?.includes("desktop") ?? image.server === LOCAL_ISO;
```

Comment:
This condition was changed to `??` beforehand, and that introduced a bug. I don't remember the exact circumstances -- but do remember changing it back and forth :)

Reply:
Right, I remember the circumstances. Weird that this triggers a warning and I didn't notice it back in the days. I will revert this change.

Reply:
I reverted the change, but please do investigate a way to remove the warning. It usually suggests that the thing could (and should) be written in a better way.